components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -181,7 +307,16 @@ service Policies {
     option (chef.automate.api.policy).action = "get";
     option (chef.automate.api.iam.policy).resource = "iam:roles:{id}";
     option (chef.automate.api.iam.policy).action = "iam:roles:get";
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "roles";
+		};
   };
+  /*
+  Delete a role
+
+  This will cascade delete any statements that only contain this role.
+  If the resulting policy has no statements, it is also deleted.
```

Comment:
I would prefer not to say "cascade delete" here; not everyone knows that jargon. Also, it is not consistent because the second line is also talking about a cascade delete yet suggests it is doing something different.
My suggestion:
```
Delete a specified role and removes it from any statements that may have been using it.
If such a statement has no other associated actions, the statement is deleted as well.
Similarly, if that statement removal results in a policy with no other statements,
that policy is removed as well.
```